ERROR - 2025-04-13 09:32:06 --> mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `artikel...', 0)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `artikel...')
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `artikel...')
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `artikel...', Array, false)
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(863): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(77): CodeIgniter\Model->update('1', Array)
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->edit('1')
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-04-13 09:32:06 --> CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away
[Method: POST, Route: admin/artikel/edit/1]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `artikel` SET `judul` = :judul:, `isi` = :isi:, `status` = :status:, `slug` = :slug:, `gambar` = :gambar:
WHERE `artikel`.`id` IN :artikel.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Artikel.php(77): CodeIgniter\Model->update('1', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->edit('1')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 09:32:06 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `artikel` SET `judul` = \'Artikel Pertama\', `isi` = \'Lorem Ipsum adalah contoh teks atau dummy dalam industri percetakan dan penataan huruf atau typesetting. Lorem Ipsum telah menjadi standar contoh teks sejak tahun 1500an, saat seorang tukang cetak yang tidak dikenal mengambil sebuah kumpulan teks dan mengacaknya untuk menjadi sebuah buku contoh huruf.\', `status` = \'publish\', `slug` = \'Artikel-Pertama\', `gambar` = NULL
WHERE `artikel`.`id` IN (\'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `artikel` SET `judul` = \'Artikel Pertama\', `isi` = \'Lorem Ipsum adalah contoh teks atau dummy dalam industri percetakan dan penataan huruf atau typesetting. Lorem Ipsum telah menjadi standar contoh teks sejak tahun 1500an, saat seorang tukang cetak yang tidak dikenal mengambil sebuah kumpulan teks dan mengacaknya untuk menjadi sebuah buku contoh huruf.\', `status` = \'publish\', `slug` = \'Artikel-Pertama\', `gambar` = NULL
WHERE `artikel`.`id` IN (\'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `artikel` SET `judul` = :judul:, `isi` = :isi:, `status` = :status:, `slug` = :slug:, `gambar` = :gambar:
WHERE `artikel`.`id` IN :artikel.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Artikel.php(77): CodeIgniter\Model->update('1', [...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->edit('1')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 09:32:06 --> [Caused by] mysqli_sql_exception: MySQL server has gone away
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('UPDATE `artikel` SET `judul` = \'Artikel Pertama\', `isi` = \'Lorem Ipsum adalah contoh teks atau dummy dalam industri percetakan dan penataan huruf atau typesetting. Lorem Ipsum telah menjadi standar contoh teks sejak tahun 1500an, saat seorang tukang cetak yang tidak dikenal mengambil sebuah kumpulan teks dan mengacaknya untuk menjadi sebuah buku contoh huruf.\', `status` = \'publish\', `slug` = \'Artikel-Pertama\', `gambar` = NULL
WHERE `artikel`.`id` IN (\'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `artikel` SET `judul` = \'Artikel Pertama\', `isi` = \'Lorem Ipsum adalah contoh teks atau dummy dalam industri percetakan dan penataan huruf atau typesetting. Lorem Ipsum telah menjadi standar contoh teks sejak tahun 1500an, saat seorang tukang cetak yang tidak dikenal mengambil sebuah kumpulan teks dan mengacaknya untuk menjadi sebuah buku contoh huruf.\', `status` = \'publish\', `slug` = \'Artikel-Pertama\', `gambar` = NULL
WHERE `artikel`.`id` IN (\'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `artikel` SET `judul` = \'Artikel Pertama\', `isi` = \'Lorem Ipsum adalah contoh teks atau dummy dalam industri percetakan dan penataan huruf atau typesetting. Lorem Ipsum telah menjadi standar contoh teks sejak tahun 1500an, saat seorang tukang cetak yang tidak dikenal mengambil sebuah kumpulan teks dan mengacaknya untuk menjadi sebuah buku contoh huruf.\', `status` = \'publish\', `slug` = \'Artikel-Pertama\', `gambar` = NULL
WHERE `artikel`.`id` IN (\'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `artikel` SET `judul` = :judul:, `isi` = :isi:, `status` = :status:, `slug` = :slug:, `gambar` = :gambar:
WHERE `artikel`.`id` IN :artikel.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Artikel.php(77): CodeIgniter\Model->update('1', [...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->edit('1')
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
ERROR - 2025-04-13 09:35:39 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `artikel...', Array, false)
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(863): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(77): CodeIgniter\Model->update('1', Array)
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->edit('1')
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `artikel...', Array, false)
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(1027): CodeIgniter\Model->doUpdate(Array, Array)
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(863): CodeIgniter\BaseModel->update(Array, Array)
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(77): CodeIgniter\Model->update('1', Array)
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->edit('1')
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-04-13 09:35:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: admin/artikel/edit/1]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `artikel` SET `judul` = :judul:, `isi` = :isi:, `status` = :status:, `slug` = :slug:, `gambar` = :gambar:
WHERE `artikel`.`id` IN :artikel.id:', [...], false)
 3 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 4 SYSTEMPATH\BaseModel.php(1027): CodeIgniter\Model->doUpdate([...], [...])
 5 SYSTEMPATH\Model.php(863): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\Artikel.php(77): CodeIgniter\Model->update('1', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->edit('1')
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
ERROR - 2025-04-13 09:35:56 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2025-04-13 09:35:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: admin/artikel/]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
ERROR - 2025-04-13 09:36:15 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2025-04-13 09:36:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: admin/artikel/]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
ERROR - 2025-04-13 09:36:21 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2025-04-13 09:36:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: admin/artikel/]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Artikel.php(38): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->admin_index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
ERROR - 2025-04-13 09:36:31 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Home.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Home->index()
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\app\Controllers\Home.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\Lab7Web\Praktikum_3 _Part2\lab11_ci\ci4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-04-13 09:36:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`
ORDER BY `tanggal` DESC', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(15): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-04-13 09:40:07 --> ErrorException: Undefined variable $kategori
[Method: GET, Route: artikel]
in APPPATH\Views\artikel\index.php on line 8.
 1 APPPATH\Views\artikel\index.php(8): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $kategori', 'C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\app\\Views\\artikel\\index.php', 8)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\app\\Views\\artikel\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('artikel/index', [], true)
 5 APPPATH\Controllers\Artikel.php(17): view('artikel/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 09:47:59 --> ErrorException: Undefined array key "slug"
[Method: GET, Route: artikel]
in APPPATH\Views\artikel\index.php on line 17.
 1 APPPATH\Views\artikel\index.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "slug"', 'C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\app\\Views\\artikel\\index.php', 17)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\app\\Views\\artikel\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('artikel/index', [], true)
 5 APPPATH\Controllers\Artikel.php(27): view('artikel/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 09:48:10 --> ErrorException: Undefined array key "slug"
[Method: GET, Route: artikel]
in APPPATH\Views\artikel\index.php on line 17.
 1 APPPATH\Views\artikel\index.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "slug"', 'C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\app\\Views\\artikel\\index.php', 17)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\app\\Views\\artikel\\index.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('artikel/index', [], true)
 5 APPPATH\Controllers\Artikel.php(27): view('artikel/index', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
DEBUG - 2025-04-13 09:54:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 09:54:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 09:54:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:05:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:06:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:13:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:19:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:23:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:26:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:30:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:33:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-04-13 11:37:23 --> Error: Class "Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 39.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:39:12 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:41:25 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:41:27 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:41:27 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:41:28 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:45:57 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:48:27 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:48:28 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
CRITICAL - 2025-04-13 11:49:41 --> Error: Class "App\Controllers\Parsedown" not found
[Method: GET, Route: artikel/Revolusi-AI-dan-IoT]
in APPPATH\Controllers\Artikel.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Artikel->view('Revolusi-AI-dan-IoT')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\Lab7Web\\Praktikum_3 _Part2\\lab11_ci\\ci4\\public\\index.php')
DEBUG - 2025-04-13 11:54:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:55:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:57:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:58:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 11:59:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:00:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:01:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:03:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:04:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:06:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:07:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:10:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:15:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:18:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:18:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:20:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-04-13 12:20:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
